components/ip2location/ip2location.app.mjs

```diff
@@ -1,11 +1,26 @@
-export default {
-  type: "app",
-  app: "ip2location",
-  propDefinitions: {},
-  methods: {
-    // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
-    },
-  },
-};
+import { axios } from "@pipedream/platform";
+
+export default {
+  type: "app",
+  app: "ip2location",
+  description: "Query IP address using IP2Location API. You can sign up for a trial key at [here](https://www.ip2location.com/register?id=1005).",

```

Comment:
Remove `at`:
...`You can sign up for a trial key at [here](https://www.ip2location.com/register?id=1005).`